engine/config_integrations/README.md

```diff
@@ -0,0 +1,19 @@
+# Contribute the new Integration to OnCall
```

Comment:
Let's add the reference to developer quickstart to help people understand how to test their changes
https://github.com/grafana/oncall/blob/dev/DEVELOPER.md#developer-quickstart